packages/reporter-vue/src/commands/CommandRow.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="command-row" :class="{ [size]: true, ...$attrs.class }"
```

Comment:
I think `$attrs.class` is merged automatically on the root node's class

Reply:
No, that behavior was removed in 3.x. Correct me if I'm wrong.

Reply:
Oh maybe, since there is fragments now